src/i18n/en.json

```diff
@@ -74,6 +74,12 @@
 	"DROPDOWN_LIST": {
 		"LABEL": "Listbox"
 	},
+	"FILE_UPLOADER": {
+		"CLOSE_BUTTON": "Close button",
```

Comment:
`CLOSE_` is a bit misleading here? Isn't it actually `REMOVE_`? same for title.

Reply:
well the svg class is `"x--file-close"`, thats why I called it `CLOSE_`